src/NLog/LayoutRenderers/Wrappers/ReplaceNewLinesLayoutRendererWrapper.cs

```diff
@@ -64,11 +60,9 @@ public string Replacement
             set
             {
                 _replacement = value;
-                _replaceWithNewLines = value?.IndexOf('\n') >= 0;
```

Comment:
Since special logic has been removed, then it can be changed to normal property:
```
public string Replacement { get; set; } = " ";
```
And remove the old backing-field `_replacement`.